### PR TITLE
Replaces Travis with GitHub Actions

### PR DESCRIPTION
**Pull request by gastaldi**
This removes the Travis descriptor and builds using GitHub actions. It also closes #751 by building daily.

This is the recommended approach as in https://jekyllrb.com/docs/continuous-integration/github-actions/

Travis has enforced some limits recently (see https://www.theregister.com/2020/11/02/travis_ci_pricng/).

### CODE REVIEW

File: .github/workflows/github-pages.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  push:
+    branches:
+      - develop
+
+jobs:
+  github-pages:
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: helaili/jekyll-action@2.0.4
```
Comment (gsmet) [inline .github/workflows/github-pages.yml:16]: Isn't there an official GH one?

File: .github/workflows/github-pages.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Build and deploy to GitHub Pages
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+  push:
+    branches:
+      - develop
```
Comment (gsmet) [inline .github/workflows/github-pages.yml:9]: I don't see PRs being tested? Travis was doing that IIRC.

File: .github/workflows/github-pages.yml (hunk 1 above)
Comment (gastaldi) [reply .github/workflows/github-pages.yml:16]: No, if you look at the [official docs](https://jekyllrb.com/docs/continuous-integration/github-actions/) you'll see that's the recommended action

File: .github/workflows/github-pages.yml (hunk 2 above)
Comment (gastaldi) [reply .github/workflows/github-pages.yml:9]: Ah yes, I'll have a look

Comment (gastaldi) [reply .github/workflows/github-pages.yml:9]: Preferably it would be nice if PRs could deploy to surge.sh to preview how the website would look like

**Comment by gastaldi** [comment]
Moving to a branch in the same repository

**Pull request closed without merging**